src/Native/Common/utilcode.h

```diff
@@ -0,0 +1,40 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+#include "daccess.h"
```

Comment:
It would look better to add this `CommonMacros.h/.inl` instead of introducing a new file for these few functions.

Reply:
Done